app/src/main/java/org/grapheneos/pdfviewer/PdfViewer.java

```diff
@@ -282,6 +282,11 @@ private void renderPage(final boolean lazy) {
         mWebView.evaluateJavascript(lazy ? "onRenderPage(true)" : "onRenderPage(false)", null);
     }
 
+    private void documentOrientationChanged(final int orientationDegreesOffset) {
+        mWebView.evaluateJavascript(String.format("onDocumentOrientationChanged(\"%s\")",
```

Comment:
This should be a member variable on the Java side in order to save and restore the state, with it exposed via the Channel class like the page number and zoom level. I also want to avoid dynamically creating any JavaScript strings for various reasons.

Reply:
Done.